### PR TITLE
Add CocoaPods documentation percentage

### PR DESCRIPTION
**Pull request by erichoracek**
This shield displays the percentage of a Pod's API that is documented as returned by the CocoaPods metrics API. It can be quite useful when figuring out if you'd like to bring in a third-party dependency.

The entire list of metrics is available [here](http://metrics.cocoapods.org/api/v1/pods/AFNetworking). This  endpoint could easily be expanded to support more metrics as needed.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -2028,6 +2028,32 @@ cache(function(data, match, sendBadge, request) {
...
+cache(function(data, match, sendBadge, request) {
+  var spec = match[1];  // eg, AFNetworking
+  var format = match[2];
+  var apiUrl = 'http://metrics.cocoapods.org/api/v1/pods/' + spec;
+  var badgeData = getBadgeData('pod', data);
+  badgeData.colorscheme = null;
+  request(apiUrl, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
```
Comment (espadrine) [inline server.js]: Can you add a `return;` statement below this line, please?

File: server.js [hunk 2]
```diff
@@ -2028,6 +2028,32 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+camp.route(/^\/cocoapods\/metrics\/doc-percent\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var spec = match[1];  // eg, AFNetworking
+  var format = match[2];
+  var apiUrl = 'http://metrics.cocoapods.org/api/v1/pods/' + spec;
+  var badgeData = getBadgeData('pod', data);
+  badgeData.colorscheme = null;
```
Comment (espadrine) [inline server.js]: Setting `colorscheme` to null is pointless here. Could you remove this line, please?

**Comment by espadrine** [comment]
Thanks a lot for this valuable contribution!

**Comment by ViccAlexander** [comment]
This is awesome. would love to see a badge for cocoapods downloads, now that new metrics are out.

**Comment by espadrine** [comment]
I didn't receive a mail that this was updated ☹

**Pull request merged**